client.go

```diff
@@ -658,24 +642,28 @@ func (c *Client) confirmRequest(request *Request) {
 func (c *Client) runRepliesConsumer(inChan *amqp.Channel) error {
 	queue, err := inChan.QueueDeclare(
 		c.replyToQueueName,
-		c.queueDeclareSettings.Durable,
-		c.queueDeclareSettings.DeleteWhenUnused,
-		c.queueDeclareSettings.Exclusive,
+		// Durable needs to be true since we must not be exclusive.
+		true,
+		// Auto-delete cannot be true since we must support reconnects. Instead
+		// we use the x-expires arg per default.
+		false,
+		// Exclusive must be false since we must support reconnects.
```

Comment:
Can we elaborate this part, why can't a queue be exclusive if using reconnects? Since it will be left on the broker for the old connection and we re-use the queue name?

Reply:
An exclusive queue can only be used by the connection that declares it.